notes/decentralization-autonomy-organisation-brekke.md

```diff
@@ -0,0 +1,68 @@
+# Keynote Decentralization, Autonomy and Organization - jaya klara brekke
+
```

Comment:
Could we provide a bit of context about where this keynote took place ...

Reply:
sure it is in the third line:

=>  Radical Friends. #HausDerKunst #GoetheInstitut DAO Summit for Decentralisation of Power and Resources in the Artworld

Thats the name of the event

Reply:
OK, i would suggest moving that up somewhere prominent e.g. the first line of the whole piece. Also add a date etc.